### PR TITLE
Net little fixes

### PR DESCRIPTION
**Pull request by G10h4ck**

### CODE REVIEW

File: libretroshare/src/pqi/p3peermgr.cc [hunk 1]
```diff
@@ -1239,11 +1239,11 @@ bool p3PeerMgrIMPL::UpdateOwnAddress( const sockaddr_storage& pLocalAddr,
 	sockaddr_storage_copy(pExtAddr, extAddr);
 	sockaddr_storage_ipv6_to_ipv4(extAddr);
 
-#ifdef PEER_DEBUG
+//#ifdef PEER_DEBUG
```
Comment (sehraf) [inline libretroshare/src/pqi/p3peermgr.cc:1242]: i think you want to revert these changes :wink:

Comment (G10h4ck) [reply libretroshare/src/pqi/p3peermgr.cc:1242]: nope, those messages are not printed often, and i need them because some user has recently reported that their RS miss some IP change, with this message enabled the console output will be helpful for that case
